### PR TITLE
Component/scatter plot/add labels

### PR DESCRIPTION
**Pull request by alanhermanns**

### CODE REVIEW

File: src/components/graphs/Scatterplot.js [hunk 1]
```diff
@@ -32,16 +32,27 @@ const Scatterplot = ({ data, xMax, yMax }) => {
     const svg = select(svgRef.current);
     const { width, height } = dimensions || wrapperRef.current.getBoundingClientRect();
     if(!dimensions) return;
+    
+
+    const removeLabelText = (localThis, args) => {
```
Comment (JMoneymaker) [inline src/components/graphs/Scatterplot.js]: localThis isn't super clear to me...

File: src/components/graphs/Scatterplot.js [hunk 2]
```diff
@@ -54,33 +65,57 @@ const Scatterplot = ({ data, xMax, yMax }) => {
...
+    svg.select('.y-label')
+      .append('text')
+      .attr('transform', 'rotate(-90)')
+      .attr('y', -50 + yScale(yMax) / 10)
+      .attr('x', 0 - yMax / 1.5)
+      .attr('dy', '1em')
+      .style('text-anchor', 'middle')
+      .text(yLabel);
+
+    // const zoomBehavior = zoom()
```
Comment (JMoneymaker) [inline src/components/graphs/Scatterplot.js:127]: I think we can just delete all the zoomBehavior stuff.

**Review by JMoneymaker — CHANGES_REQUESTED** [review]
Could you change the This thing? Sorry, it's really confusing to me.

**Review by licarpen — CHANGES_REQUESTED** [review]
what is localThis?

**Review by licarpen — APPROVED** [review]

**Review by yaycub — APPROVED** [review]

**Review by JMoneymaker — APPROVED** [review]

**Pull request merged**